### PR TITLE
Clean up dumpLocations a bit

### PR DESCRIPTION
**Pull request by chenglou**

### CODE REVIEW

**Comment by chenglou** [comment]
Before:
Screen Shot 2021-04-13 at 6 34 35 AM
After:
Screen Shot 2021-04-13 at 6 35 23 AM

Longer but maybe more readable.
Didn't use real json because this is for debugging clarity.

File: src/EditorSupportCommands.ml [hunk 1]
```diff
@@ -1,108 +1,49 @@
...
-             | None -> []
-             | Some s -> [("hover", Json.String s)]
-           in
-           let uriLocOpt =
-             References.definitionForLoc ~pathsForModule:package.pathsForModule
-               ~file ~getUri:(State.fileForUri state)
-               ~getModule:(State.fileForModule state ~package)
-               loc
-           in
-           let def, skipZero =
```
Comment (chenglou) [inline src/EditorSupportCommands.ml:49]: Removed skipZero. Didn't make a different on my test file and not sure if important enough

File: src/EditorSupportCommands.ml [hunk 2]
```diff
@@ -1,108 +1,49 @@
...
-               (* Skip if range is all zero, unless it's a module *)
-               let skipZero =
-                 (not locIsModule) && posIsZero loc.loc_start
-                 && posIsZero loc.loc_end
-               in
-               let range = ("range", rangeOfLoc loc) in
-               ( [
-                   ( "definition",
-                     Json.Object
-                       (match uriIsCurrentFile with
```
Comment (chenglou) [inline src/EditorSupportCommands.ml:66]: Removed this check too

File: src/EditorSupportCommands.ml (hunk 1 above)
Comment (cristianoc) [reply src/EditorSupportCommands.ml:49]: It's needed for correctness, with PPXs etc.
But, for debugging it might even be better to keep it.

**Comment by chenglou** [comment]
Removed the line col api for dumpLocation too since we can just use the actual hover/definition api if we need a single location

**Pull request merged**